tests/yatai/client/test_bento_repository_api.py

```diff
@@ -1,150 +1,109 @@
 # pylint: disable=redefined-outer-name
 import logging
 import os
-import signal
-import subprocess
 import sys
-import time
 import uuid
 
-import psutil  # noqa # pylint: disable=unused-import
+import docker
+import psutil
 import pytest
 
 import bentoml
-import docker
 from bentoml.configuration import LAST_PYPI_RELEASE_VERSION
-from bentoml.exceptions import InvalidArgument
 from bentoml.saved_bundle.loader import load_from_dir
+from bentoml.utils.tempdir import TempDirectory
 from bentoml.yatai.client import get_yatai_client
 from bentoml.yatai.deployment.docker_utils import ensure_docker_available_or_raise
-from bentoml.yatai.db.stores.label import _validate_labels
+from tests.yatai.local_yatai_service import wait_until_container_ready
 
 logger = logging.getLogger('bentoml.test')
 
-
-def test_validate_labels_fails():
-    with pytest.raises(InvalidArgument):
-        _validate_labels(
-            {'this_is_a_super_long_key_name_it_will_be_more_than_the_max_allowed': 'v'}
-        )
-    with pytest.raises(InvalidArgument):
-        _validate_labels({'key_contains!': 'value'})
-    with pytest.raises(InvalidArgument):
-        _validate_labels({'key': 'value-contains?'})
-    with pytest.raises(InvalidArgument):
-        _validate_labels({'key nop': 'value'})
-    with pytest.raises(InvalidArgument):
-        _validate_labels({'key': '1', 'key3@#!$': 'value'})
-    with pytest.raises(InvalidArgument):
-        _validate_labels({'key': 'cant_end_with_symbol_'})
-
-
-def test_validate_labels_pass():
-    _validate_labels({'long_key_title': 'some_value', 'another_key': "value"})
-    _validate_labels({'long_key-title': 'some_value-inside.this'})
-    _validate_labels({'create_by': 'admin', 'py.version': '3.6.8'})
-
-
-def wait_until_container_ready(container_name, check_message, timeout_seconds=120):
-    docker_client = docker.from_env()
-
-    start_time = time.time()
-    while True:
-        time.sleep(1)
-        container_list = docker_client.containers.list(filters={'name': container_name})
-        logger.info("Container list: " + str(container_list))
-        if not container_list:
-            # Raise timeout, if exceeds timeout limit
-            if time.time() - start_time > timeout_seconds:
-                raise TimeoutError(f'Get container: {container_name} timed out')
-            else:
-                continue
-
-        assert (
-            len(container_list) == 1
-        ), f'should be exact one container with name {container_name}'
-
-        if check_message in container_list[0].logs():
-            break
+if sys.platform == "darwin" or not psutil.POSIX:
+    pytest.skip(
+        "Requires docker, skipping test for Mac OS on Github Actions",
+        allow_module_level=True,
+    )
 
 
-@pytest.fixture(scope='module')
-def yatai_server_container():
+@pytest.fixture(scope='session')
+def local_yatai_service_url():
```

Comment:
just `yatai_service_url `?  "local" is a bit misleading here since it's not what we call "local yatai" in the code

Reply:
can we just reuse the same `local_yatai_service_container` function from `tests/yatai/local_yatai_service.py`?